webpack.prod.config.js

```diff
@@ -1,9 +1,19 @@
+const fs = require('fs');
+const dotenv = require('dotenv');
 const path = require('path');
 const { createConfig } = require('@edx/frontend-build');
 const CopyPlugin = require('copy-webpack-plugin');
 
 const config = createConfig('webpack-prod');
 
+// Overides development variables, this is required because this line
+// https://github.com/nelc/frontend-build/blob/open-release/palm.nelp/lib/plugins/paragon-webpack-plugin/ParagonWebpackPlugin.js#L20-L22
+// is always executed no matter the environment
+const envConfig = dotenv.parse(fs.readFileSync('.env'));
+Object.keys(envConfig).forEach(k => {
+  process.env[k] = envConfig[k];
+});
+
```

Comment:
```suggestion
dotenv.config({
  path: path.resolve(process.cwd(), '.env'),
  override:  true,
});
```
I think that could do the same, 
inspired in this frontend build way 
https://github.com/openedx/frontend-build/blob/v12.8.42/config/webpack.prod.config.js#L28-L30
but with override
https://github.com/openedx/frontend-build/blob/v12.8.42/config/webpack.prod.config.js#L28-L30

You could tried...

Reply:
the override feature was included in the 15.0.0 version we are in the  8.6

Reply:
So the way you did it, is the way.